plugins/frontend-search/README.md

```diff
@@ -62,6 +62,10 @@ Available search contexts are:
 
 If you want to have another context, open an Issue and tell us!
 
+## Changing the fallback search behaviour
+
+You can set the fallback search engine to DuckDuckGo by setting `FRONTEND_SEARCH_FALLBACK_SEARCH_ENGINE='duckduckgo'` in your `~/.zshrc` file.
```

Comment:
Would you mind to add the available options in this documentation, sharing that by default is using google, but it can uses `duckduckgo` if added in `FRONTEND_SEARCH_FALLBACK_SEARCH_ENGINE`, please? It will be better knowing in this readme than having to look at the code

Reply:
No problem, I've updated the readme to include this.